src/components/pages/Offerers/OfferersContainer.jsx

```diff
@@ -51,9 +51,9 @@ export const mapDispatchToProps = (dispatch, ownProps) => ({
     let apiPath = createApiPath(searchKeyWords)
 
     if (apiPath.includes('?')) {
-      apiPath = `${apiPath}&page=${pageParams}`
+      apiPath = `${apiPath}&page=${pageParams}&is_active=true`
```

Comment:
```suggestion
      apiPath = `${apiPath}&page=${pageParams}&isActive=true`
```

Normalement c'est du camelCase les paramètres envoyés par url